src/modules/75-cf/components/NoData/NoData.tsx

```diff
@@ -35,21 +35,23 @@ export const NoData: React.FC<NoDataProps> = ({
   return (
     <Layout.Vertical className={css.centerAlignItems} spacing="xxlarge" width={width || 470} {...props}>
       <img src={imageURL} width={320} height={220} />
-      <Heading
-        className={css.centerAlign}
-        level={2}
-        font={{ variation: FontVariation.H4 }}
-        width={546}
-        color={Color.GREY_600}
-      >
+      <Heading className={css.centerAlign} level={2} font={{ variation: FontVariation.H4 }} color={Color.GREY_600}>
         {message}
       </Heading>
       {description && (
-        <Text className={css.centerAlign} font={{ variation: FontVariation.BODY1 }} width={546} color={Color.GREY_600}>
+        <Text className={css.centerAlign} font={{ variation: FontVariation.BODY1 }} color={Color.GREY_600}>
           {description}
         </Text>
       )}
-      {buttonText ? <Button intent="primary" text={buttonText} width={buttonWidth} onClick={onClick} /> : null}
+      {buttonText ? (
+        <Button
+          intent="primary"
+          variation={ButtonVariation.TERTIARY}
+          text={buttonText}
+          width={buttonWidth}
+          onClick={onClick}
+        />
+      ) : null}
```

Comment:
Might as well bring this inline with the style of the part above it.
```suggestion
      {buttonText && (
        <Button
          intent="primary"
          variation={ButtonVariation.TERTIARY}
          text={buttonText}
          width={buttonWidth}
          onClick={onClick}
        />
      )}
```

Reply:
Cool 👍